lib/index.js

```diff
@@ -166,7 +166,7 @@ app.get('/config', (req, res) => {
        }
 
         let displayName = response.display_name;
-        if (!displayName || response.trim().length === 0) {
+        if (!displayName) {
```

Comment:
Well, sleeping while coding isn't that helpful. I think I wanted to trim `displayName` not `response`

```suggestion
        if (!displayName || displayName.trim().length === 0) {
```